### DISCUSSION TITLE
GitHub Android - Notifications show `@username requested your review on...` yet they didn't

### DISCUSSION THREAD

**Discussion by setchy** [community/community#48752] [Mobile]
### Select Topic Area

Bug

### Body

I've experience this issue for as long as I can remember... sharing with hopes it may be a simple bug-fix

On GitHub Android, I frequently will receive a notification for PRs which  says "@username requested your review on...".  Yet when I click on the notification and view the actual PR, I'm am in fact not listed as a reviewer or being requested to review.

Attached is an example illustrating this quirk 🥴 

![Screenshot_20230227-185528~2](https://user-images.githubusercontent.com/386277/221737424-94638310-5a7f-4549-a9d5-15ed03972b28.png)
![Screenshot_20230227-185539~2](https://user-images.githubusercontent.com/386277/221737427-f5626cfc-1178-4232-86ca-800839c6bdec.png)
![Screenshot_20230227-201536~2](https://user-images.githubusercontent.com/386277/221737429-ded0594e-6847-41da-a635-75e9c92e2563.png)

**Comment by HenSquared** [comment]
Hey @setchy ! thanks for the feedback. Does your repository use auto-assign reviewers? Or are reviewers manually assigned? 

https://docs.github.com/en/organizations/organizing-members-into-teams/managing-code-review-settings-for-your-team#about-auto-assignment

**Reply by setchy** [reply]
@HenSquared - I'm not sure sorry.

**Reply by HenSquared** [reply]
No problem. 

I will discuss with team and try to understand this bug and hopefully a fix later! Thanks for reporting.

**Reply by Chinnaphat152** [reply]
> Hey @setchy ! thanks for the feedback. Does your repository use auto-assign reviewers? Or are reviewers manually assigned? 
> 
> https://docs.github.com/en/organizations/organizing-members-into-teams/managing-code-review-settings-for-your-team#about-auto-assignment
>

**Reply by setchy** [reply]
> > Hey @setchy ! thanks for the feedback. Does your repository use auto-assign reviewers? Or are reviewers manually assigned? 
> > 
> > https://docs.github.com/en/organizations/organizing-members-into-teams/managing-code-review-settings-for-your-team#about-auto-assignment
> > 
> 
> 

Not sure sorry @Chinnaphat152

**Reply by jsoref** [reply]
The review isn't coming from https://github.com/renovatebot/renovate/blob/main/.github/CODEOWNERS, and it appears to be manual, so it's probably some form of team assignment.

https://github.com/renovatebot/renovate/pull/20652#event-8621363171
